### PR TITLE
Update rogers2020a_2.md

### PR DESCRIPTION
**Pull request by timchen0618**

### CODE REVIEW

**Comment by timchen0618** [comment]
I am not sure if you think the summary is too long. 
Do you think it's okay if I remove the "BERT Embedding" part and the "Pretraining" and "Fine-Tuning" part?

**Comment by Shiviagarw** [comment]
Hi, Yes the summary is long. You can shorten it by converting bullets to paragraph. For instance convert the bullet point under "Syntactic Knowledge" into paragraph and tyr to implement it in this way for all sub headings.

Yes it should be ok if you remove BERT Embedding" part and the "Pretraining" and "Fine-Tuning" part .
Also please add one experiment conducted using BERT .

Other things looks fine.

**Comment by Shiviagarw** [comment]
Hi, Yes the summary is long. You can shorten it by converting bullets to paragraph. For instance convert the bullet point under "Syntactic Knowledge" into paragraph and tyr to implement it in this way for all sub headings.

Yes it should be ok if you remove BERT Embedding" part and the "Pretraining" and "Fine-Tuning" part .
Also please add one experiment conducted using BERT .

Other things looks fine.

**Comment by timchen0618** [comment]
I am not sure if converting bullet points into paragraphs will do any good because the contents are basically the same? It's just it "looks" longer, but bullet points offers better readability I think?

**Comment by timchen0618** [comment]
Also I don't know what "experiment conducted using BERT" is.

**Comment by Shiviagarw** [comment]
Hi Tim,
It's up to you. If you feel the bullet points will give better readability
then let it remain the same. And by experiments I meant the future work
suggestion , which you have already added. The summary looks good to go.



Shivi Agarwal
(She/her/hers)
Master of Science Computer Science
graduation year(2021-2022) University Of Texas, Austin
***@***.***
Cell # 5125865037

**Comment by timchen0618** [comment]
Thank you! I'll think about your suggestions.

File: _summaries/rogers2020a_2.md [hunk 1]
```diff
@@ -1,19 +1,90 @@
...
+- Should focus on how to reason over the information contained in BERT
+- Learn what BERT actually do during test time instead of probing
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+- BERT has learned a great amount of syntactic, semantic and even world knowledge, but the key to perform well is how to extract and reason with these information. 
+- Different layers and attention heads of BERT seem to learn different kind of information during pretraining.
+- BERT is overparameterized and does not fully utilize the parameters it has; thus it can be compressed without much performance loss.
```
Comment (Shiviagarw) [inline _summaries/rogers2020a_2.md:90]: The summary looks good and all important points have been covered as well.

**Review by Shiviagarw — APPROVED** [review]
The summary looks good and all important points have been covered as well.

**Review by zhaoyue-zephyrus — APPROVED** [review]
